fl4health/checkpointing/client_module.py

```diff
@@ -34,31 +40,37 @@ def __init__(
             pre_aggregation (CheckpointModuleInput, optional): If defined, this checkpointer (or sequence of
                 checkpointers) is used to checkpoint models based on their validation metrics/losses **BEFORE**
                 server-side aggregation. Defaults to None.
-            post_aggregation (CheckpointModuleInput, optional], optional): If defined, this checkpointer (or sequence
+            post_aggregation (CheckpointModuleInput, optional): If defined, this checkpointer (or sequence
```

Comment:
Is there an instance where the post-aggregation client model is different from the global model?